includes/admin/class-wcmp-admin.php

```diff
@@ -772,6 +773,14 @@ private function printDeliveryDate(DeliveryOptions $delivery_options): void
         }
     }
 
+    /**
+     * @return string
+     */
+    public static function getDefaultCarrier(): string
+    {
+        return WCMP()->setting_collection->getByName(WCMP_Settings::SETTING_DEFAULT_CARRIER);
```

Comment:
kan die setting ook leeg zijn? Misschien nog een fallback erachter naar `WCMP_Data::DEFAULT_CARRIER`

Reply:
Hij kan niet leeg zijn, er is altijd iets geselecteerd (bpost)

Reply:
Ja maar voordat die settings pagina een keer is opgeslagen?

Reply:
goed punt